libraries/AP_RangeFinder/RangeFinder_Backend.h

```diff
@@ -55,6 +55,9 @@ class AP_RangeFinder_Backend
     // in metres relative to the body frame origin
     const Vector3f &get_pos_offset() const { return state.pos_offset; }
 
+    // return system time of last successful read from the sensor (used for DPTH logging)
```

Comment:
I would remove the comment for what it's used, that tends to get outdated, specially in libraries.